docs/RFCS/README.md

```diff
@@ -14,7 +14,7 @@ What is a major change?
 A change is "major" and requires a RFC when any of the following
 conditions apply:
 
-- completing the change will cost you more than a week of work.
+- completing the change will cost you more than two weeks of work.
```

Comment:
This needs to be synchronized with the "how to contribute to cockroachdb" doc page, and also bounced back to Nate who was coordinating the definition of the contribution process.